aptos-move/framework/src/natives/cryptography/bulletproofs.rs

```diff
@@ -112,6 +116,13 @@ fn native_verify_batch_range_proof(
     debug_assert!(_ty_args.is_empty());
     debug_assert!(args.len() == 6);
 
+    let ctx = context.extensions().get::<BulletproofContext>();
```

Comment:
Why we do not do this for `native_verify_range_proof`?

Reply:
Is there a way to do it in a backwards compatible manner? 

Right now, `native_verify_range_proof` is enabled on `mainnet`.

Reply:
You can have a feature flag or use gas feature version and add a check there? The only issue if people already build with these as then mainnet applications are broken: but we can find out with replay?